test/clj/chronograph/domain/user_test.clj

```diff
@@ -61,3 +62,25 @@
                        :created-at         (time/now)
                        :updated-at         (time/now)}
                retrieved-user))))))
+
+(deftest organizations-test
+  (testing "Returns the list of organizations a user belongs to"
+    (let [user (factories/create-user)
+          organization1 (factories/create-organization (:users/id user))
+          organization2 (factories/create-organization (:users/id user))]
+      (with-transaction [tx db/datasource]
+        (let [user-organizations (user/organizations tx user)]
+          (is (= 2 (count user-organizations)))
+          (is (= #{(:organizations/id organization1)
+                   (:organizations/id organization2)}
+                 (set (map :organizations/id user-organizations))))))))
```

Comment:
If `user/organizations` ever returns duplicate values, this coercion into a `set` will obscure that error.

A sorted collection of IDs would present a better assert, IMO.

```clojure
(= (sort (map :organizations/id [organization1 organization2]))
   (sort (map :organizations/id user-organizations)))
```

Reply:
If `user/organizations` return duplicate values, the second set would have only one element, so the assertion would still fail, no?

Reply:
If `user-organizations` becomes `[org1 org1 org2 org2]`, then coercing it to a set would hide this problem.